core/src/test/scala/org/apache/spark/rdd/DoubleRDDSuite.scala

```diff
@@ -18,10 +18,17 @@
 package org.apache.spark.rdd
 
 import org.scalatest.FunSuite
+import org.scalatest.Matchers._
 
 import org.apache.spark._
 
 class DoubleRDDSuite extends FunSuite with SharedSparkContext {
+  test("sum") {
+    sc.parallelize(Seq.empty[Double]).sum() should be(0.0 +- 0.0001)
```

Comment:
I would just use `assert` here as int he rest of the file. Also I think you can assert exact equality in these cases. It really should be `3.0` in the last instance, precisely.